.github/workflows/continuous-integration.yml

```diff
@@ -32,31 +32,21 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - name: Check out WALA sources
-        uses: actions/checkout@v3
-      - name: Cache Goomph
-        uses: actions/cache@v3
-        with:
-          path: ~/.goomph
-          key: ${{ runner.os }}-goomph-${{ hashFiles('build.gradle') }}
-          restore-keys: ${{ runner.os }}-goomph-
```

Comment:
I removed this as I think it's not recommended when using the new `setup-gradle` action:

https://github.com/gradle/actions/blob/main/docs/setup-gradle.md#incompatibility-with-other-caching-mechanisms